app/views/banners/_form.html.haml

```diff
@@ -0,0 +1,28 @@
+%h3
+
+.grid-x.grid-padding-y
+  .cell.small-12.medium-6
+    = f.label :visible, 'Visible'
+    = f.check_box :visible, value: banner.visible
+.grid-x.grid-padding-y
+  .cell
+    = f.label :body, 'Body'
+    = f.hidden_field :body, id: :banner_body
+    %trix-editor(input='banner_body')
+.grid-x.grid-padding-y
+  .cell.small-12.medium-6
+    = f.label :created_at, 'Created At'
+    = f.text_field :created_at, value: banner.created_at, disabled: true, class: 'small-12 medium-4'
```

Comment:
Pretty sure value: is not needed here (or elsewhere)

Reply:
Removed.